requirements/tests.txt

```diff
@@ -7,3 +7,5 @@ sphinx==2.2.1
 sphinx-rtd-theme==0.4.3
 git+https://github.com/behave/behave@v1.2.7.dev1
 allure-behave==2.8.10
+
+python-vlc==1.1.2
```

Comment:
Was this meant to be added to the tests requirements?

Reply:
Yup. One of the tests actually uses it, I had it fail when it wasn't installed. Since it's an optional dependency at runtime, it has to be specified here.